src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/InformationGetter.kt

```diff
@@ -0,0 +1,223 @@
+            "Your Tasty Mithr",
+            "§3§lSolo",
+            "§fRift Dimension",
+            "§d᠅ §fGemstone",
+            "§2᠅ §fMithril",
+            "Revenant Horror",
+            "Tarantula Broodfa",
+            "Sven Packm",
+            "Voidgloom Seraph",
+            "Inferno Demo",
+            "Combat XP",
+            "Flight Duration:",
+            "§a✌ §",
+            "Points: ",
+            "Challenge:",
+            *dungeonClassList.toTypedArray(),
+            "§cLocked",
+            "§fCleanup§7:",
+            "§fPasting§7: ",
+            "§fBarn Pasting§7:",
+            "§6Year ",
+            "§7Waiting for",
+            "§7your vote...",
+            "Pelts: §5",
+            "Tracker Mob Location:",
+            "§7Boss: §",
+            "§7Damage Soaked:",
+            "§6Kill the Magmas:",
+            "Time Left: §b",
+            "Current Item:",
+            "Effigies: ",
+            "§cThe boss is reforming!",
+            "§7Boss Health:",
+            "Find tickets on the",
+            "ground and bring them",
+            "to the raffle box",
+            "Tickets: §a",
+            "Pool: §6"
+        )
+
+        unknownLines = sidebarLines.filter { line -> !knownLines.any { line.trim().contains(it) } }
+
+        // filter empty lines
+        unknownLines = unknownLines.filter { it.isNotBlank() }
+
+        // remove objectives kuudra
+        unknownLines = unknownLines.filter { sidebarLines.nextAfter("§fObjective") != it }
+        unknownLines = unknownLines.filter {
+            sidebarLines.nextAfter("§fObjective", 2) != it && !extraObjectiveKuudraLines.contains(it)
+        }
+
+        // remove objectives
+        val objectiveLine =
+            ScoreboardData.sidebarLinesFormatted.firstOrNull { it.startsWith("Objective") } ?: "Objective"
+        unknownLines = unknownLines.filter { sidebarLines.nextAfter(objectiveLine) != it }
+        unknownLines =
+            unknownLines.filter { sidebarLines.nextAfter(objectiveLine, 2) != it && !extraObjectiveLines.contains(it) }
+
+        // remove wind compass
+        unknownLines = unknownLines.filter { sidebarLines.nextAfter("§9Wind Compass") != it }
+
+        // Remove jacobs contest
+        for (i in 1..3)
+            unknownLines = unknownLines.filter { sidebarLines.nextAfter("§eJacob's Contest", i) != it }
+
+        // Remove slayer
+        unknownLines = unknownLines.filter { sidebarLines.nextAfter("Slayer Quest", 1) != it }
+        unknownLines = unknownLines.filter { sidebarLines.nextAfter("Slayer Quest", 2) != it }
+
+        // remove voting lines
+        val votedLine = sidebarLines.firstOrNull { it.startsWith("§6Year ") } ?: "§6Year "
+        for (i in 1 until 6) {
+            unknownLines = unknownLines.filter { sidebarLines.nextAfter(votedLine, i) != it }
+        }
+
+        // remove trapper mob location
+        unknownLines = unknownLines.filter { sidebarLines.nextAfter("Tracker Mob Location:", 1) != it }
```

Comment:
For the logic that happens in this line you could write a utils function `removeNextAfter(` to have fewer filter and nextAfter calls, fewer code duplication

Reply:
the original line on which we call nextAfter() will absolutely not exist here anymore
thats why I call sidebarLines.nextAfter so we know the line after here "Tracker Mob Location"

and then we can filter that out